frontend/packages/dev-console/src/components/pipelines/pipeline-resource/pipelineResource-utils.ts

```diff
@@ -27,7 +27,7 @@ export const createPipelineResource = (
   const resourceName = `${type}-${getRandomChars(6)}`;
   const pipelineResource: K8sResourceKind = {
     apiVersion: 'tekton.dev/v1alpha1',
-    kind: 'PipelineResource',
+    kind: referenceForModel(PipelineResourceModel),
```

Comment:
@vikram-raj This should be left alone or changed to:
```suggestion
    kind: PipelineResourceModel.kind,
```

It's odd though. I tested the code and although it creates a `kind` with value `tekton.dev~v1alpha1~PipelineResource`, the POST request to create the resource works.